src/reducers/versions.tsx

```diff
@@ -212,6 +213,10 @@ export const actions = {
     return (payload: { selectedPath: string; versionId: number }) =>
       resolve(payload);
   }),
+  toggleExpandedPath: createAction('TOGGLE_EXPANDED_PATH', (resolve) => {
+    return (payload: { selectedPath: string; versionId: number }) =>
```

Comment:
I suggest a name other than `selectedPath` to avoid confusion with `version.selectedPath`. Maybe just `path`?